### DISCUSSION TITLE
Suggestions on script automatic silent update

### DISCUSSION THREAD

**Discussion by violenttamper** [Tampermonkey/tampermonkey#1242] [Q&A]
请问，tampermonkey扩展在更新大脚本时会弹出窗口，怎么设置更新不提醒？如果没有这个功能能否考虑加入这个功能，我认为这可能会是个好功能。谢谢

**Comment by Geeyun-JY3** [comment]
用英文啊……

**Reply by SapientSi** [reply]
没事的，用中文没事 而且用中文 还有文化输出作用，老用英文 中文怎么发扬光大，不过有些论坛歧视汉语，上次我在一个论坛里用中文，他们竟然把我的号封杀了￼￼

**Reply by Geeyun-JY3** [reply]
提高交流效率而已，别扯这些有的没的。

**Reply by violenttamper** [reply]
之前看到一论坛里英文交流的几个人，一看都是中国人，哈哈。我觉得对于英文不太好的人，想要方便反馈，打中文也没什么问题。开发者，用户都可以使用工具翻译，谈不上什么效率不效率的。用户使用产品，发现问题进行反馈，开发者看到反馈，查看问题，并最终解决问题。开发者有责任去理解用户的诉求，用户在使用产品的时候也有义务在发现问题的时候尽量去反馈问题，以此让产品更加完善。

**Comment by violenttamper** [comment]
Excuse me, the monkey extension will pop up when updating a large script. How can I set the update not to remind?
If you don't have this function, can you consider adding this function? I think it might be a good function. thank you
